packages/flame_texturepacker/lib/src/texture_packer_atlas.dart

```diff
@@ -25,18 +25,22 @@ class TexturePackerAtlas {
   /// If [useOriginalSize] is true, the sprites loaded will be use original size
   /// instead of the packed size. For animation sprites, load with origin size
   /// is recommended for smooth result.
+  ///
+  /// If [prefix] is not specified, the atlas file will be expected to under
+  /// a folder named "images" under [Flame.assets]
   static Future<TexturePackerAtlas> load(
     String path, {
     bool fromStorage = false,
     Images? images,
     bool useOriginalSize = true,
+    String? prefix,
```

Comment:
```suggestion
    String prefix = 'images',
```

Set the default here (so that the user can see it in their IDE), and set the internal methods to have this argument as required and non-nullable.

Reply:
Maybe name it `assetsPrefix` instead, so that it isn't mixed up with the image cache prefix.

Reply:
Ok got it, I have made the `_fromStorage` also not accept the parameter just for simplicity sakes. 

It should be good now

Reply:
Don't worry about the commits, they are all squashed on merge :)